### ISSUE TITLE
Bug: Arguments of multi word languages throws error

### ISSUE THREAD

**Issue by joshpetit** [kraftwerk28/gtranslate.nvim#4]
Hey! I'm super glad I found this plugin, really great job with it.

There is a small bug that doesn't let you translate to/from languages that span more than 1 word. For example:

```vim
:Translate English Haitian Creole
:Translate English "Haitian Creole"
"Bad "to" language.
```

I'm assuming it's cause it's only taking in "Haitian" and not "Haitian Creole"

**Comment by kraftwerk28** [comment]
Fixed in a3ce65cd, now multi-word language names are joined with underscores.

**Comment by joshpetit** [comment]
Awesome stuff! Thanks.
